### PR TITLE
[WIP] Limit the pitch to -50 degrees

### PR DESCRIPTION
**Pull request by fredj**
Work in progress, don't merge

### CODE REVIEW

File: src/js/GaCesium.js [hunk 1]
```diff
@@ -59,6 +60,16 @@ var GaCesium = function(map, gaPermalink, gaLayers, gaGlobalOptions, $q) {
     return cesiumViewer;
   };
 
+  var limitCameraPitch = function() {
+    var angle = Cesium.Math.toRadians(-50);
+    console.log(this.pitch, angle);
```
Comment (gjn) [inline src/js/GaCesium.js]: Debug artefact?

File: src/js/GaCesium.js [hunk 2]
```diff
@@ -59,6 +60,16 @@ var GaCesium = function(map, gaPermalink, gaLayers, gaGlobalOptions, $q) {
...
+    if (this.pitch > angle) {
+      this.setView({
+        pitch: angle
+      })
```
Comment (gjn) [inline src/js/GaCesium.js]: Missing semicolon.

**Comment by fredj** [comment]
I've updated the code: the restriction is now applied immediately.
(but still don't merge, the results are not good)

**Comment by cedricmoullet** [comment]
Will not be merged

**Pull request closed without merging**